librubyfmt/src/format_prism.rs

```diff
@@ -1637,6 +1637,13 @@ fn format_block_parameters_node(
     ps: &mut dyn ConcreteParserState,
     block_parameters_node: prism::BlockParametersNode,
 ) {
+    // Exit early if there's no params
+    if node_list_is_empty(&block_parameters_node.locals())
+        && block_parameters_node.parameters().is_none()
+    {
+        return;
+    }
+
```

Comment:
This is just to match behavior of the Ripper implementation, which doesn't render empty block params (`||`)